IoTuring/ClassManager/consts.py

```diff
@@ -1,2 +1,2 @@
-ENTITIES_PATH = "../Entity/Deployments/"
-WAREHOUSES_PATH = "../Warehouse/Deployments/"
+ENTITIES_PATH = "Entity/Deployments"
+WAREHOUSES_PATH = "Warehouse/Deployments"
```

Comment:
@richibrics I would change this file to something like this: 
```suggestion
KEY_ENTITY = "entity"
KEY_WAREHOUSE = "warehouse"

PATHS = {
    KEY_ENTITY: "Entity/Deployments",
    KEY_WAREHOUSE: "Warehouse/Deployments"
}
```

Than you can use it like this:

```
ecm = ClassManager(KEY_ENTITY)
ecm.ListAvailableClasses()
```

You can just easily add new keys and paths here.

Reply:
Okay we can leave it with separated keys and add a new key in the future for custom entity. It's okay for me !